src/components/BlogPostHeader.vue

```diff
@@ -16,7 +16,7 @@
             blogPost.publishedAt.toDateString()
           }}
         </li>
-        <li class="list-inline-item">
+        <li class="list-inline-item d-none">
```

Comment:
I was thinking about that. hiding something (with a comment or style like that), because it will be needed later is always very tempting. I am guilty myself of this in this project.

Right now website is in more mature state. Blog with live comment section will not be added anytime soon, so knowing that, I would actually suggest of removing those dead sections.

This code will not be lost, as it is in the git and original design. But if we will allow to stay it there... it will affect readability of the code. So, please remove not used part.